PhysicsTools/NanoAOD/python/jetsAK4_CHS_cff.py

```diff
@@ -246,44 +246,55 @@
-                                  "Jet_chHEF","Jet_chEmEF","Jet_leptonPtRelInv","isEle","isMu","isOther","Jet_mass","Jet_ptd"]),
-    variables = cms.PSet(
-    Jet_pt = cms.string("pt*jecFactor('Uncorrected')"),
-    Jet_mt = cms.string("mt*jecFactor('Uncorrected')"),
-    Jet_eta = cms.string("eta"),
-    Jet_mass = cms.string("mass*jecFactor('Uncorrected')"),
-    Jet_ptd = cms.string("userFloat('ptD')"),
-    Jet_leadTrackPt = cms.string("userFloat('leadTrackPt')"),
-    Jet_vtxNtrk = cms.string("userInt('vtxNtrk')"),
-    Jet_vtxMass = cms.string("userFloat('vtxMass')"),
-    Jet_vtx3dL = cms.string("userFloat('vtx3dL')"),
-    Jet_vtx3deL = cms.string("userFloat('vtx3deL')"),
-    Jet_vtxPt = cms.string("userFloat('vtxPt')"),
-    Jet_leptonPtRel = cms.string("userFloat('leptonPtRelv0')"),
-    Jet_leptonPtRelInv = cms.string("userFloat('leptonPtRelInvv0')*jecFactor('Uncorrected')"),
-    Jet_leptonDeltaR = cms.string("userFloat('leptonDeltaR')"),
-    Jet_neHEF = cms.string("neutralHadronEnergyFraction()"),
-    Jet_neEmEF = cms.string("neutralEmEnergyFraction()"),
-    Jet_chHEF = cms.string("chargedHadronEnergyFraction()"),
-    Jet_chEmEF = cms.string("chargedEmEnergyFraction()"),
-    isMu = cms.string("?abs(userInt('leptonPdgId'))==13?1:0"),
-    isEle = cms.string("?abs(userInt('leptonPdgId'))==11?1:0"),
-    isOther = cms.string("?userInt('leptonPdgId')==0?1:0"),
+    variables = cms.VPSet(
+        cms.PSet( name = cms.string("Jet_pt"), expr = cms.string("pt*jecFactor('Uncorrected')")),
+        cms.PSet( name = cms.string("Jet_eta"), expr = cms.string("eta")),
+        #cms.PSet( name = cms.string("rho"), expr = cms.string("")),
+        cms.PSet( name = cms.string("Jet_mt"), expr = cms.string("mt*jecFactor('Uncorrected')")),
+        cms.PSet( name = cms.string("Jet_leadTrackPt"), expr = cms.string("userFloat('leadTrackPt')")),
+        cms.PSet( name = cms.string("Jet_leptonPtRel"), expr = cms.string("userFloat('leptonPtRelv0')")),
+        cms.PSet( name = cms.string("Jet_leptonDeltaR"), expr = cms.string("userFloat('leptonDeltaR')")),
+        cms.PSet( name = cms.string("Jet_neHEF"), expr = cms.string("neutralHadronEnergyFraction()")),
+        cms.PSet( name = cms.string("Jet_neEmEF"), expr = cms.string("neutralEmEnergyFraction()")),
+        cms.PSet( name = cms.string("Jet_vtxPt"), expr = cms.string("userFloat('vtxPt')")),
+        cms.PSet( name = cms.string("Jet_vtxMass"), expr = cms.string("userFloat('vtxMass')")),
+        cms.PSet( name = cms.string("Jet_vtx3dL"), expr = cms.string("userFloat('vtx3dL')")),
+        cms.PSet( name = cms.string("Jet_vtxNtrk"), expr = cms.string("userInt('vtxNtrk')")),
+        cms.PSet( name = cms.string("Jet_vtx3deL"), expr = cms.string("userFloat('vtx3deL')")),
+        #cms.PSet( name = cms.string("Jet_numDaughters_pt03"), expr = cms.string("")),
```

Comment:
@nurfikri89 : can you please check why those variables were present in the ordering, but not in the variable definition ? what am I missing ?